db/migrate/20161006152332_add_default_category_to_restaurants.rb

```diff
@@ -0,0 +1,5 @@
+class AddDefaultCategoryToRestaurants < ActiveRecord::Migration[5.0]
+  def change
+    change_column_default :restaurants, :category, 'Other'
```

Comment:
Don't even know where this comes from?.. Should be :string? But I can't change it directly in the migration file, right?